### PR TITLE
Allow option to specify scheme for usar avatar url. Defaults to http

### LINKED ISSUE #39: User image URL causes HTTPS mixed content warning when user image is embedded in a page encrypted with SSL
I use the user image returned by this provider as a current user avatar. The URL for this resource is hardcoded to "http". When this image is embedded in a page that has "https", the browser throws a mixed content warning.

We should allow the option to specify the scheme. Either "https" or "http" with it defaulting to "http"

### PR DESCRIPTION
**Pull request by jabberfest**
This PR adds the option to specify the scheme for the url of the user's image. It defaults to "http" when none is specified.

I'm not sure how to go about testing this. The repo currently has no tests. Any feedback would be appreciated.

This should fix https://github.com/ueberauth/ueberauth_facebook/issues/39

### CODE REVIEW

**Comment by yordis** [comment]
@jabberfest I would prefer to default it to HTTPS.

**Comment by jabberfest** [comment]
I agree. Should we even allow the option to set it to HTTP? I don't see any practical reason. The fix can be as simple as changing the "http" to "https".

I can make that change to the PR if you approve.

**Comment by yordis** [comment]
@jabberfest I dont see why somebody would use HTTP over HTTPs.

**Comment by jabberfest** [comment]
I reverted by initial changes, and just changed the http to https. Also got travis green for what it's worth.

File: mix.exs [hunk 1]
```diff
@@ -27,7 +27,7 @@ defmodule Ueberauth.Facebook.Mixfile do
     [{:ueberauth, "~> 0.4"},
      {:oauth2, "~> 0.9"},
 
-     {:credo, "~> 0.8", only: [:dev, :test]},
+     {:credo, "~> 0.8.10", only: [:dev, :test]},
```
Comment (doomspork) [inline mix.exs:30]: For future reference, this change is unnecessary.

**Pull request merged**